src/subscript/params2csv/params2csv.py

```diff
@@ -103,14 +183,12 @@ def main() -> None:
         )  # if key is repeated, keep the last one.
         transposed = paramtable.set_index("key").transpose()
         if args.filenamecolumnname in transposed.columns:
-            print(
-                "Column name "
-                + args.filenamecolumnname
-                + " was already in "
-                + parameterfilename
-                + ", not writing"
+            logger.info(
+                "Column name %s was already in %s, not writing",
+                args.filenamecolumnname,
+                parameterfilename,
             )
-            print(
+            logger.info(
                 "this filename into CSV output. Use --filenamecolumnname to avoid this."
             )
```

Comment:
Could these two log statements be collapsed to one?

Reply:
Yes, it's definitely a good point. :+1: